bindings/ros/README.md

```diff
@@ -2,69 +2,105 @@
 # ROS Wrapper for the ADI ToF library
 
 ## Overview
-This ROS package facilitates DEPTH and IR data acquisition and processing for the Analog Devices depth cameras.
+This ROS package facilitates depth and IR data acquisition and processing for the Analog Devices depth cameras.
 
-## On Target
-  
-1.1 **Install the recommended [ROS distribution](http://wiki.ros.org/Distributions) for your operating system**
-  - [ROS Install page](http://wiki.ros.org/ROS/Installation)
-  - On target choose: [ROS Melodic Distro](http://wiki.ros.org/melodic/Installation/Ubuntu) and install the ROS-Base (Bare Bones) package
-**Install the ADI ToF SDK library**
-  - [Install SDK dependencies](https://github.com/analogdevicesinc/ToF/blob/6c7fb376aeec73a21ab177adf297c5781bcbd544/doc/linux/build_instructions.md#installing-the-dependencies)
+## Installation
 
-  1.2 Download the SDK from the repository \
-  1.3 Copy the following files to the specific location after installing the ROS package:
-  
+- **Install the recommended [ROS distribution](http://wiki.ros.org/Distributions) for your operating system**
+  - [ROS Install page](http://wiki.ros.org/ROS/Installation)
+- **Make sure you have these ROS packages installed before building and running the examples**
+  - [rviz](http://wiki.ros.org/rviz)\
+    Run the command below for each package, replacing ROSDISTRO with the name of the ROS distribution you are using and PACKAGE with the name of the needed package.
   ```console
-  sudo cp <libs folder direcory>/libtofi_config.so $ROS_ROOT/../../lib/.
-  sudo cp <libs folder direcory>/libtofi_compute.so $ROS_ROOT/../../lib/.
-  sudo cp -r <config folder directory> $HOME/.ros/
+  sudo apt install ros-ROSDISTRO-PACKAGE
   ```
-  where "libs folder direcory" is the location of the folder where the depth_compute files are stored and the "config folder directory" is the directory where the neccessary config files are strored for the camera. \
-
-
-  1.4 After this, build the SDK, as well as enable ROS package building 
-
+- **Install the ADI ToF SDK library**
+  - [Install SDK dependencies](https://github.com/analogdevicesinc/aditof_sdk/blob/6c7fb376aeec73a21ab177adf297c5781bcbd544/doc/linux/build_instructions.md#installing-the-dependencies)
```

Comment:
I would use a link that contains master and not a specific commit SHA like this one: 6c7fb376aeec73a21ab177adf297c5781bcbd544

Reply:
Changed to new reference